traffic_ops/ort/atstccfg/config/config.go

```diff
@@ -140,6 +140,9 @@ func GetCfg() (Cfg, error) {
 	if toPass == "" {
 		toPass = os.Getenv("TO_PASS")
 	}
+	if toPass == "" {
+		toPass = os.Getenv("TO_PASSWORD")
+	}
```

Comment:
Because along with `TO_USER` and `TO_URL`, `TO_PASSWORD` is used by the `to_access` package, the `to_access.sh` script, the `compare` and `genConfigFileRoutes.py` tools, and some other things which make it semi-standard. But `TO_PASS` was there first, so I didn't want to break compatibility with existing systems by changing it. So instead I added the more normal one and switched the documentation to only mention that, and then later we might be able to get rid of the one-off `TO_PASS` support.

Reply:
That makes sense. Even though it's in the documentation, these lines could probably use a comment so people know that `TO_PASSWORD` is the preferred one.